v-next/hardhat-mocha-test-runner/test/index.ts

```diff
@@ -0,0 +1,54 @@
+import assert from "node:assert/strict";
+import { describe, it, afterEach } from "node:test";
+
+import { HardhatError } from "@ignored/hardhat-vnext-errors";
+import {
+  assertRejectsWithHardhatError,
+  useFixtureProject,
+} from "@nomicfoundation/hardhat-test-utils";
+
+describe("Hardhat Mocha plugin", () => {
+  describe("Success", () => {
+    useFixtureProject("test-project");
+
+    afterEach(async () => {
+      const { _resetGlobalHardhatRuntimeEnvironment } = await import(
```

Comment:
I think that instead of this, you could use `@ignored/hardhat-vnext/hre` and initialize the HRE explicitly. That way, we don't use the global instance in our tests, and we don't need to export the reset function.

Would you mind doing that? I think we could also add this to the guidelines, as it sounds like something that every plugin would have to deal with.